src/list.tsx

```diff
@@ -122,17 +122,24 @@ interface CommonProps {
     extra?: React.ReactNode;
     dataAttributes?: DataAttributes;
     disabled?: boolean;
+    navigable?: boolean; // enables chevron on custom elements with text
```

Comment:
if you rename this prop to "withChevron" you can remove the comment and it is self explanatory

Reply:
Done!